### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by yuukiiwa**
Release 3.0.0

- new DSL2 syntax
- Azure bug fix
- DNA structural variant detection
- RNA modification and fusion detection
- output demultiplexed `fast5`

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanoseq/tree/master/.github/CONTRIBUTING.md)
    - [x] If necessary, also make a PR on the nf-core/nanoseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated  (including new tool citations and authors/contributors).

`test_nobc_nodx` --> `test_nobc_nodx_stringtie` because running `bambu` in parallel for both `test_nobc_nodx` and `test_nobc_nodx_noaln` exceeds the GitHub Action memory limit, causing `segmentation fault core dump` on GitHub Action. Here is the local run of test_nobc_nodx (with bambu):
nanoseq_bambu_test_fastq

### CODE REVIEW

File: subworkflows/local/dna_variant_calling.nf
Comment (christopher-hakkaart) [inline subworkflows/local/dna_variant_calling.nf:41]:
```suggestion
    ch_sv_calls = Channel.empty()
    sniffles_version = Channel.empty()
```

File: nextflow_schema.json
Comment (christopher-hakkaart) [inline nextflow_schema.json:94]:
```suggestion
                    "description": "Whether to trim the barcodes from the output sequences in the FastQ files from Guppy basecaller."
```

File: nextflow_schema.json
Comment (christopher-hakkaart) [inline nextflow_schema.json:138]:
```suggestion
                    "description": "Demultiplex fast5 files with demux_fast5.",
```

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]
It looks pretty good. Most of the comments are moving/correcting the version reporting for sniffles and medaka, couple of small typos, and a .view() left over from debugging (I assume).

Really great!

File: docs/output.md
Comment (yuukiiwa) [reply docs/output.md:46]: I have changed this to fast5

File: docs/output.md
Comment (yuukiiwa) [reply docs/output.md:168]: I have changed this to `SAMPLE`. (I can't commit the suggestions because I don't have write access to `dev`)

File: docs/output.md
Comment (yuukiiwa) [reply docs/output.md:53]: Yes, I have changed this to fast5

File: nextflow_schema.json
Comment (yuukiiwa) [reply nextflow_schema.json:138]: I have changed this accordingly

File: nextflow_schema.json
Comment (yuukiiwa) [reply nextflow_schema.json:94]: I have fixed this typo

File: subworkflows/local/dna_variant_calling.nf
Comment (yuukiiwa) [reply subworkflows/local/dna_variant_calling.nf:18]: I have added medaka_version

File: subworkflows/local/dna_variant_calling.nf
Comment (yuukiiwa) [reply subworkflows/local/dna_variant_calling.nf:41]: I have added `sniffles_version`

File: workflows/nanoseq.nf
Comment (yuukiiwa) [reply workflows/nanoseq.nf:338]: I have removed `ch_medaka_version`

File: workflows/nanoseq.nf
Comment (yuukiiwa) [reply workflows/nanoseq.nf:339]: I have removed `ch_sniffles_version`

File: subworkflows/local/dna_variant_calling.nf
Comment (yuukiiwa) [reply subworkflows/local/dna_variant_calling.nf:39]: I have added `medaka_version = MEDAKA_VARIANT.out.versions`

File: subworkflows/local/dna_variant_calling.nf
Comment (yuukiiwa) [reply subworkflows/local/dna_variant_calling.nf:47]: I have added `sniffles_version = SNIFFLES.out.versions`

File: subworkflows/local/dna_variant_calling.nf
Comment (yuukiiwa) [reply subworkflows/local/dna_variant_calling.nf:53]: I have added `sniffles_version` and `medaka_version`

File: workflows/nanoseq.nf
Comment (yuukiiwa) [reply workflows/nanoseq.nf:341]: I have added the two lines

File: docs/usage.md
Comment (yuukiiwa) [reply docs/usage.md:16]: Thank you for the fix!! I have changed this with your sentence

**Review by christopher-hakkaart — COMMENTED** [review]
I think all the changes I suggested are addressed. Are there any last minute changes/bugs you wanted addressed in dev before we approve the merge?

**Pull request closed without merging**
